jsk_fetch_robot/jsk_fetch_startup/config/install_udev.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+jsk_fetch_startup=$(builtin cd "`dirname "${BASH_SOURCE[0]}"`"/.. > /dev/null && pwd)
+
+IFS=':' read -r -a prefix_paths <<< "$CMAKE_PREFIX_PATH"
+current_prefix_path="${prefix_paths[0]}"
```

Comment:
`IFS` and `current_prefix_path` are not used in this script. Do we really need this?

Reply:
Thank you for your advice.

I removed unnecessary variables.